README.md

```diff
@@ -1 +1,67 @@
-# LeaderBoard
\ No newline at end of file
+# LeaderBoard
+> Leaderboard list app, using webpack and ES6 features
+
+## Built With
+- HTML
+- CSS
+- Javascript
+- Linters
+- Webpack
+
+
+## Live Demo
+Will update soon.
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps. (NPM)
+1. Open your terminal
+2. Navigate to the directory where you want your project cloned
+3. Clone the project using `git clone` or just copy the command below:
+```  
+   git clone https://github.com/asharanjith/LeaderBoard.git
+```   
+4. Open the project with your IDE. I use [Visual Studio Code](https://code.visualstudio.com/download)
+5. Run `npm install` to get the dependencies
+   not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+6. Run the webpack server using this command:
+``` 
+   npm run build
+```  
+  not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+### Prerequisites
+- Having an IDE .
+- Having Git installed and configured.
+
+### Setup
+- Clone the repository using Git.
+- Open the project folder in your IDE.
+- Get the dependecies
+
+
+👤 **Author**
+
+- GitHub: [@asharanjith](https://github.com/asharanjith)
+- Twitter: [@asha_cep](https://twitter.com/asha_cep)
+- LinkedIn: [asha-vijayan](www.linkedin.com/in/ashavijayan)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Your issues page is broken 😨 Kindly fix this link to make your readme descriptive 😊